### ISSUE TITLE
will the book ever get finished? 

### ISSUE THREAD

**Issue by ErikEngerd** [luksa/kubernetes-in-action-2nd-edition#32]
ordered the book years ago. Will this book ever be finished? Very disappointing to have to wait for years for this book to be published. I liked the content so far but the MEAP hasn't received any updates so for for the last year.

**Comment by SergeiCherevko** [comment]
+1

**Comment by knrc** [comment]
It's being worked on again, with some announcements coming up for those of you who have been patient.  I'll chase up Manning to see what is happening with those.

**Comment by yoshikakbudto** [comment]
It is the path that matter, not destination :)

**Comment by mnuman** [comment]
I have also MEAP'ed the book over a year ago and until now I am quite happy with its contents. I am hoping it will indeed be finished in Spring 2025, but there right now 17 out of 31 chapters are available. The remaining ones appear to be very interesting, so I hope @luksa will find the time to complete these as well!

**Comment by mnuman** [comment]
> It's being worked on again, with some announcements coming up for those of you who have been patient. I'll chase up Manning to see what is happening with those.

I'd lover to hear whether there are any updates from Manning, @knrc ? Complete and thorough is better than fast ...

**Comment by ErikEngerd** [comment]
indeed, the last chapters seem very interesting, which is why I am still holding on to this book after all these years.

**Comment by hlloreda** [comment]
> It's being worked on again, with some announcements coming up for those of you who have been patient. I'll chase up Manning to see what is happening with those.

I maybe missed it but has anything been announced?

**Comment by mnuman** [comment]
> > It's being worked on again, with some announcements coming up for those of you who have been patient. I'll chase up Manning to see what is happening with those.
> 
> I maybe missed it but has anything been announced?

Can’t see any newer chapters beyond 17 have been published - I lost faith it was going to happen and returned my MEAP. If it stilk gets published, I would seriously consider buying it but for now I am not waiting on it. Just my two cents.

**Comment by duongleh** [comment]
Looks like the book has finally been released
https://www.manning.com/books/kubernetes-in-action-second-edition
https://www.amazon.com/Kubernetes-Action-Second-Marko-Luk%C5%A1a/dp/1617297615/

**Comment by mnuman** [comment]
> Looks like the book has finally been released https://www.manning.com/books/kubernetes-in-action-second-edition https://www.amazon.com/Kubernetes-Action-Second-Marko-Luk%C5%A1a/dp/1617297615/

Yes, finally! I ordered the book in August 2023 and when I contacted the people at Manning in February 2025, I was allowed to swap it. However, I noticed that it was still going to be published in 2026 and re-ordered it and just yesterday I finally received a shipment notice from Manning - but I will only cheer when I have the book in my hands.
However, despite all the delays and bumps in the process, I still think this really is a remarkable and very useful book, so I am looking forward to receiving it!

**Comment by ErikEngerd** [comment]
I ordered it in 2021. Did not receive a shipment notice yet. They are not using first come first serve apparently.

**Comment by ErikEngerd** [comment]
Book is released
